CHANGELOG.md

```diff
@@ -96,6 +96,7 @@ Consider migrating to `stream-chat-android-ui-components` or `stream-chat-androi
 - Added support for Giphy command.
 - Added message pinning to the list of message options
 - Added pinned message UI
+- Added a checkbox to the `MessageComposer` component, that allows to display a thread message in the parent channel.
```

Comment:
I see that [unlike iOS](https://github.com/GetStream/stream-chat-swift/blob/main/CHANGELOG.md#-changed), actually you arent referencing to the pull request that provided the change. 
Treat this as a comment, rather than an action to take within this PR :)

Reply:
I'm moving this discussion to Slack